google/ggrc-core
# Revert 7245 refresh revisions

# Issue description

The original PR created delete revisions for all objects (even existing ones). 

# Steps to test the changes

Run refresh revisions and check that there are no "deleted" revisions created for objects that still exist in our db

# Solution description

Revert of the PR that caused the regression

# Sanity checklist




# PR Review checklist
